public/susi-chatbot.js

```diff
@@ -33,223 +33,203 @@ let botbuilderUserMessageTextColor = '#ffffff';
-    'background-color',
-    botbuilderUserMessageBackground,
-  );
-  $('head').append(
-    $(
-      '<style>.susi-comment-body-container-user:after { border-color:' +
-        `transparent transparent ${botbuilderUserMessageBackground} ${botbuilderUserMessageBackground} !important` +
-        '}</style>',
+  getComputedStyle(
+    document.querySelector(
+      '.susi-comment-by-user .susi-comment-body-container',
     ),
+  )['background-color'] = botbuilderUserMessageBackground;
+
+  const head = document.querySelector('head');
+
+  head.insertAdjacentHTML(
+    'beforeend',
+    `<style>
+      .susi-comment-body-container-user:after { 
+        border-color:' transparent transparent ${botbuilderUserMessageBackground} ${botbuilderUserMessageBackground} !important'
+      }
+    </style>`,
   );
-  $('.susi-comment-by-user .susi-comment-body-container').css(
-    'color',
-    botbuilderUserMessageTextColor,
-  );
+
+  getComputedStyle(
+    document.querySelector(
+      '.susi-comment-by-user .susi-comment-body-container',
+    ),
+  )['color'] = botbuilderUserMessageTextColor;
+
   // bot message container
-  $('.susi-comment-by-susi .susi-comment-body-container').css(
-    'background-color',
-    botbuilderBotMessageBackground,
-  );
-  $('.susi-comment-by-susi .susi-comment-body-container').css(
-    'color',
-    botbuilderBotMessageTextColor,
-  );
+
+  getComputedStyle(
+    document.querySelector(
+      '.susi-comment-by-susi .susi-comment-body-container',
+    ),
+  )['background-color'] = botbuilderBotMessageBackground;
+
+  getComputedStyle(
+    document.querySelector(
+      '.susi-comment-by-susi .susi-comment-body-container',
+    ),
+  )['color'] = botbuilderBotMessageTextColor;
+
   // bot icon
-  $('.susi-launcher-button').css('background-color', botbuilderIconColor);
-  $('.susi-comment-avatar').css('background-color', botbuilderIconColor);
+  getComputedStyle(document.querySelector('.susi-launcher-button'))[
+    'background-color'
+  ] = botbuilderIconColor;
+
+  getComputedStyle(document.querySelector('.susi-comment-avatar'))[
+    'background-color'
+  ] = botbuilderIconColor;
+
   if (botbuilderIconImg) {
-    $('.susi-launcher-button').css(
-      'background-image',
-      "url('" + botbuilderIconImg + "')",
-    );
-    $('.susi-comment-avatar').css(
-      'background-image',
-      "url('" + botbuilderIconImg + "')",
-    );
+    getComputedStyle(document.querySelector('.susi-launcher-button'))[
+      'background-image'
+    ] = `url('${botbuilderIconImg}')`;
+
+    getComputedStyle(document.querySelector('.susi-comment-avatar'))[
+      'background-image'
+    ] = `url('${botbuilderIconImg}')`;
   }
 }
 
 function enableBot() {
-  $(document).ready(function() {
+  let ready = callback => {
+    if (document.readyState != 'loading') callback();
+    else document.addEventListener('DOMContentLoaded', callback);
+  };
+
+  ready(() => {
     const baseUrl = api_url + '/susi/chat.json?q=';
```

Comment:
Changes not made here, else looks good.

Reply:
I didn't saw that 
Thanks for pointing 👍